### PR TITLE
doc: Update default fport in enable storage integration API

### PR DESCRIPTION
**Pull request by Siva-7482**
#### Summary

In the enable storage integration guide of `HTTP(REST)API` section, the FPORT is mentioned as `10`. Updated the storage integration default port as `100`.
Ref: https://www.thethingsindustries.com/docs/integrations/storage/enable/

Ref: https://github.com/TheThingsIndustries/lorawan-stack-support/issues/1041#issuecomment-2034795043

#### Screenshots

NA

#### Changes

- Updated the default FPORT `100` for enabling the storage integration using APIs.

#### Notes for Reviewers

...

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by nejraselimovic — COMMENTED** [review]
LGTM based on the referenced discussion but pls wait for @adriansmares approval

**Review by adriansmares — APPROVED** [review]

**Pull request merged**
